pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -3259,17 +3260,17 @@ func (c *Cloud) getInstancesByNodeNamesCached(nodeNames sets.String) ([]*ec2.Ins
 	return instances, nil
 }
 
-func (c *Cloud) getInstancesByNodeNames(nodeNames []string) ([]*ec2.Instance, error) {
+func (c *Cloud) getInstancesByNodeNames(nodeNames []string, states ...string) ([]*ec2.Instance, error) {
```

Comment:
Just as a tip for the future... If we change the signature, it's better IMO to make it incompatible.  It helps the coder find all the places, in means the reviewer doesn't need to check that all the callsites have been changed.  But more than that... There are other PRs or people's work in flight, and we don't want to _silently_ break them.  So in this case it would be `states []string`.